client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -52,7 +52,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I install plugins?' ) }
 				answer={ translate(
-					'Yes! With the WordPress.com Business plan you can search for and install external plugins.' +
+					'Yes! With the WordPress.com Business or eCommerce plan you can search for and install external plugins.' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Yes! With the WordPress.com Business plan you can search for and install external plugins. All plans already come with a custom set of plugins tailored just for them. {{a}}Find out more about plugins{{/a}}.' )` *ES Score: 8*